polars/polars-lazy/polars-plan/src/dsl/expr.rs

```diff
@@ -278,7 +278,11 @@ impl AsRef<Expr> for AggExpr {
     }
 }
 
-/// Queries consists of multiple expressions.
+// TODO: is there any way to link to `polars_lazy::dsl` here? It's not a dependency of polars-plan (and can't be —
+// circular dependency) so I don't see any way to get a link to it in here.
+/// An abstract representation of a DataFrame column or an operation to perform on one. Queries consist of multiple
```

Comment:
Can we name this differently. Expressions can be used on aggregates and list elements as well.

In python we only say: "Expressions that can be used in various contexts.", I think we should go for that.